unstructured/ingest/connector/salesforce.py

```diff
@@ -242,9 +130,15 @@ def create_eml(self, email_json: t.Dict[str, t.Any]) -> str:
             from_email=email_json.get("FromAddress"),
             to_email=email_json.get("ToAddress"),
             textbody=email_json.get("TextBody"),
+            # TODO: This is a hack to get emails to process correctly.
+            # The HTML partitioner seems to have issues with <br> and text without tags like <p>
+            htmlbody=email_json.get("HtmlBody", "")  # "" because you can't .replace None
+            .replace("<br />", "<p>")
+            .replace("<body", "<body><p"),
```

Comment:
is the tag `<p` meant to be left open like that? i.e. not `<p>`?

Reply:
should we also be replacing the `</body>` with `</p></body>`

Reply:
Yeah. This is not pretty for sure. The open `<p` is completed by the style tags and other stuff that I can't reliably predict (unless I do a regex which may be better.

```
<body style="height: auto; min-height: auto;">
```
turns to 
```
<body><p style="height: auto; min-height: auto;">
```

The partitioning doesn't seem to care if there are </p>

Reply:
ohhh, got it, I missed the fact that the `<body` tag was also dangling. yea, not pretty, but seems to do the trick and hopefully just temporary. thanks for clarifying!